auth_otp/admin.py

```
@@ -113,14 +138,34 @@ class OTPVerificationAdmin(admin.ModelAdmin):
+        پارامترها:
+            obj: نمونه‌ای از مدل OTPVerification که فیلد رابطه‌ای `otp_request` را دارد.
+        
+        بازگشت:
+            str: مقدار `phone_number` استخراج‌شده از `obj.otp_request`.
+        """
         return obj.otp_request.phone_number
     get_phone_number.short_description = 'شماره موبایل'
     
     actions = ['deactivate_sessions']
     
     def deactivate_sessions(self, request, queryset):
```

Ruff (0.12.2)

Missing return type annotation for public function `deactivate_sessions`

Add return type annotation: `None`

(ANN201)